### PR TITLE
Plug Syncto with Cliquet docs.

### PR DESCRIPTION
**Pull request by Natim**
Kinto is nicely plugged with Cliquet docs, we should do the same with Syncto.

### CODE REVIEW

File: docs/api/resource.rst [hunk 1]
```diff
@@ -1,28 +1,275 @@
+.. _resource-endpoints:
+
 ##################
 Resource endpoints
 ##################
 
-.. _resource-endpoints:
+GET /buckets/syncto/collections/{collection}/records
```
Comment (almet) [inline docs/api/resource.rst]: We should use the http domain extension for sphinx here.

File: docs/api/resource.rst [hunk 2]
```diff
@@ -1,28 +1,275 @@
...
+A ``Last-Modified`` response header provides a human-readable (rounded to second)
+of the current collection timestamp.
+
+For cache and concurrency control, an ``ETag`` response header gives the
+value that consumers can provide in subsequent requests using ``If-Match``
+and ``If-None-Match`` headers (see :ref:`section about timestamps <server-timestamps>`).
+
+**Request**:
+
+.. code-block:: http
```
Comment (almet) [inline docs/api/resource.rst]: Use httpie and separate request and response bodies;

File: docs/api/resource.rst [hunk 3]
```diff
@@ -1,28 +1,275 @@
...
+ - keys
+ - meta
 
 The returned value is a JSON mapping containing:
 
-- ``items``: the list of records, with exhaustive attributes
+- ``data``: the list of records, with exhaustive fields;
+
+A ``Total-Records`` response header indicates the total number of records
+of the collection.
```
Comment (almet) [inline docs/api/resource.rst]: _in_ the collection

File: docs/api/resource.rst [hunk 4]
```diff
@@ -1,28 +1,275 @@
...
+
+A ``Last-Modified`` response header provides a human-readable (rounded to second)
```
Comment (almet) [inline docs/api/resource.rst]: human-readable _version_?

File: docs/api/resource.rst [hunk 5]
```diff
@@ -1,28 +1,275 @@
...
+of the current collection timestamp.
+
+For cache and concurrency control, an ``ETag`` response header gives the
+value that consumers can provide in subsequent requests using ``If-Match``
+and ``If-None-Match`` headers (see :ref:`section about timestamps <server-timestamps>`).
```
Comment (almet) [inline docs/api/resource.rst]: "the section about"

File: docs/api/resource.rst [hunk 6]
```diff
@@ -1,28 +1,275 @@
...
+    		...
+        ]
+    }
+
+
+Filtering and Sorting
+---------------------
+
+Filtering properties of the Firefox Sync protocol are exposed as well
+as the one needed by Kinto clients.
```
Comment (almet) [inline docs/api/resource.rst]: I don't understand what that means.

File: docs/api/resource.rst [hunk 7]
```diff
@@ -1,28 +1,275 @@
...
+
+
+Pagination
+----------
+
+The ``Next-Page`` header will be sent to you with the URL to fetch the
+next page. It will include defined ``_limit`` and ``_token`` values
+automatically.
+
+When the ``Next-Page`` is not present, it means that you fetched
```
Comment (almet) [inline docs/api/resource.rst]: "it means there is no more data to fetch"

File: docs/api/resource.rst [hunk 8]
```diff
@@ -1,28 +1,275 @@
...
+everything already.
+
+
+Counting
+--------
+
+Contrary to what Kinto does, the ``Total-Records`` only contains the
+number of records in the current request
```
Comment (almet) [inline docs/api/resource.rst]: nit: contained in the current

File: docs/api/resource.rst [hunk 9]
```diff
@@ -1,28 +1,275 @@
...
+`for now <https://github.com/mozilla-services/syncto/issues/43>`_.
+
+You may ask the request without the ``_limit`` parameter to get all
+the records at once.
+
+
+Polling for changes
+-------------------
+
+The ``_since`` parameter is provided as an alias for ``gt_last_modified``.
```
Comment (ametaireau) [inline docs/api/resource.rst]: maybe say what this means.

File: docs/api/resource.rst [hunk 10]
```diff
@@ -1,28 +1,275 @@
...
+---------------------
+
+Filtering properties of the Firefox Sync protocol are exposed as well
+as the one needed by Kinto clients.
+
+- ``_since`` with the ETag value to fetch changes from the previous
+  time.
+- ``_sort`` can be either ``newest`` or ``index`` or ``-last_modified`` and ``-sortindex``.
+- ``_limit`` to limit the number of items per pages (no limit by default).
+- ``ids`` to define the list of requested records.
```
Comment (leplatrem) [inline docs/api/resource.rst:116]: Oh, what is the format of this `ids` ? Shouldn't it be `in_id=` ?

File: docs/api/resource.rst [hunk 11]
```diff
@@ -1,28 +1,275 @@
...
+
+
+Pagination
+----------
+
+The ``Next-Page`` header will be sent to you with the URL to fetch the
```
Comment (leplatrem) [inline docs/api/resource.rst]: _to you_ is superflous.

File: docs/api/resource.rst [hunk 12]
```diff
@@ -1,28 +1,275 @@
...
+collection was not changed, a ``304 Not Modified`` response is returned.
+
+
+Additionnal headers
+-------------------
+
+The ``Quota-Remaining`` header is not part of the Kinto protocol yet
+but is passed through if present in Firefox Sync responses.
+
+Its value is in kB.
```
Comment (leplatrem) [inline docs/api/resource.rst]: kilo bytes

Comment (phrawzty) [inline docs/api/resource.rst]: _Kilobyte_ is one word (no space).

As an aside, this is a surprisingly tricky term! `kB` vs `KB`; `kilo` vs `kebi`, etc.
https://en.wikipedia.org/wiki/Kilobyte

File: docs/api/resource.rst (hunk 10 above)
Comment (Natim) [inline docs/api/resource.rst:116]: I have created https://github.com/mozilla-services/syncto/issues/50 about that.

File: docs/api/resource.rst (hunk 1 above)
Comment (Natim) [inline docs/api/resource.rst]: If we do it here, we should probably as well in `cliquet_docs/api/resource.rst`

Comment (Natim) [inline docs/api/resource.rst]: You are right, [I like it](http://payments.readthedocs.org/en/latest/design/api/storage.html#storage-api)

**Pull request merged**